src/components/Pivot/Pivot.scss

```diff
@@ -9,7 +9,7 @@
 .ms-Pivot {
   @include ms-font-m;
   @include ms-u-normalize;
-  height: 30px;
+  height: 40px;
```

Comment:
There are new redlines for Pivot that were implemented here along with the high contrast pass. 40px height are a part of the new design along with the indicator at the bottom and an update to the colors being used.